autoscale-api/src/main/java/com/sequenceiq/periscope/api/model/ActivityStatus.java

```diff
@@ -5,6 +5,12 @@ public enum ActivityStatus {
     METRICS_COLLECTION_SUCCESS,
     METRICS_COLLECTION_FAILED,
 
+    MANDATORY_UPSCALE,
+    MANDATORY_DOWNSCALE,
+
+    SCHEDULE_BASED_UPSCALE,
+    SCHEDULE_BASED_DOWNSCALE,
+
```

Comment:
Do we need to add SCHEDULE_BASED_DOWNSCALE, SCHEDULE_BASED_UPSCALE, MANDATORY_DOWNSCALE, MANDATORY_UPSCALE, SCALING_FLOW_UNKNOWN in ApiActivityStatus file?

Reply:
Yup, theyve been mapped